Runtimes/Supplemental/cmake/modules/FindSwiftCore.cmake

```diff
@@ -288,7 +288,10 @@ find_path(SwiftShims_INCLUDE_DIR "shims/module.modulemap" HINTS
   ${SwiftShims_INCLUDE_DIR_HINTS})
 add_library(swiftShims INTERFACE IMPORTED GLOBAL)
 set_target_properties(swiftShims PROPERTIES
-  INTERFACE_INCLUDE_DIRECTORIES "${SwiftShims_INCLUDE_DIR}/shims")
+  INTERFACE_INCLUDE_DIRECTORIES "${SwiftShims_INCLUDE_DIR}/..;${SwiftShims_INCLUDE_DIR}/shims")
+target_compile_options(swiftShims INTERFACE
+  "$<$<AND:$<NOT:$<BOOL:${BUILD_SHARED_LIBS}>>,$<COMPILE_LANGUAGE:Swift>>:SHELL:-Xcc -DSWIFT_STATIC_STDLIB>"
```

Comment:
Should the shims be setting the `SWIFT_STATIC_STDLIB`? I think that it might be better to do this from `swiftCore` as a `PUBLIC` `target_compile_options`.